src/game/GamemodeHandler.js

```diff
@@ -30,23 +30,34 @@ class GMHandlerClass {
       testGamemode: [],
     };
 
+    /**
+       The name, amount and order of the buttons used by a gamemode
+     */
+    this.buttonsUsed = {
+      knockOff: ['Super Heavy'],
+      knockOffDynamic: ['Super Heavy'],
+      knockOffRandom: ['Super Heavy'],
+      knockOffWander: ['Super Heavy'],
+      testGamemode: ['Test Ability 1', 'Test Ability 2'],
```

Comment:
Forgot to add the dangerbot gamemode buttons.

I would add them myself but I'm not sure what type of buttons that gamemode uses.